src/Deal/DealController.php

```diff
@@ -19,16 +20,19 @@ public function catchDataFromGoogleForm(Request $request, Response $response): R
     {
         $jsonFromBody = $this->getParsedJsonFromBody($request);
 
+        /** @var Event[] $eventsWithAccess */
+        $eventsWithAccess = $request->getAttribute('eventsWithAccess');
```

Comment:
Není to špatně, ale jde to lépe. Díky PHP-DI lze tyto atributy díky qutowiringu použít přímo jako atribut v metodě :exploding_head: Je to potom lépe dosažitelné a typové, jde totiž použít něco jako 
`public function catchDataFromGoogleForm(Request $request, Response $response, Event $eventWithAccess): Response`

Víc můžeš načíst tady: https://php-di.org/doc/frameworks/slim.html#controller-parameters

Reply:
bohužel ne když je to array<Event>, co s tím?